css/about.css

```diff
@@ -0,0 +1,193 @@
+
```

Comment:
One thing that will make it easier when you are creating multiple CSS pages is you can put the normalize/reset CSS in its own file too.  then you just call it before calling your about/index CSS on your HTML pages.